### PR TITLE
Nk/us2 to master

### PR DESCRIPTION
**Pull request by nkeller1**
## What functionality does this accomplish?
Completes User Story 2. 
**Description:**
Full functionality in shelters. Show Page
DB. 
Created Reviews DB
Established Foreign Key on Shelters Table
Created review model
Added reviews to Shelter Show page.

## What did you struggle on to complete?
Not sure if I am testing for edge cases. Also created an instance of a review for the shelter show page. Show we call @shelter.reviews instead?

### Test Coverage Percentage: x%00
- [ ] No Tests have been changed
- [x] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
- [ ] New Tests added for new features

## Checklist:
- [x] My code has no unused/commented out code
-  [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

### CODE REVIEW

File: app/views/shelters/show.html.erb [hunk 1]
```diff
@@ -8,3 +8,13 @@
...
+
+<h3>Reviews:</h3>
+
+<% @shelter.reviews.each do |review| %>
+<p>Title: <%= review.title %> </p>
+<p>Rating: <%= review.rating %></p>
+<p>Content: <%= review.content %> </p>
+<p>Picture:</p>
+<%= image_tag(review.picture, height: "100", width: "150", alt: "Reviewer Picture") %>
+<% end %>
```
Comment (itemniner) [inline app/views/shelters/show.html.erb]: Great!

File: db/migrate/20200204230526_add_reviews_to_shelters.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddReviewsToShelters < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :reviews, :shelter, foreign_key: true
+  end
+end
```
Comment (itemniner) [inline db/migrate/20200204230526_add_reviews_to_shelters.rb]: nice job!

File: db/schema.rb [hunk 3]
```diff
@@ -26,6 +26,17 @@
     t.index ["shelter_id"], name: "index_pets_on_shelter_id"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "title"
+    t.integer "rating"
+    t.string "content"
+    t.string "picture"
```
Comment (itemniner) [inline db/schema.rb]: one thing we have to do is add the picture as an "optional picture" by giving it a default
